src/lib/Components/Bidding/Context/TimeOffsetProvider.tsx

```diff
@@ -1,41 +1,35 @@
+import { TimeOffsetProviderQuery } from "__generated__/TimeOffsetProviderQuery.graphql"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
 import * as PropTypes from "prop-types"
 import * as React from "react"
-import { metaphysics } from "../../../metaphysics"
-
-interface SystemResults {
-  data: {
-    system: {
-      time: {
-        unix: number
-      }
-    }
-  }
-}
+import { fetchQuery, graphql } from "relay-runtime"
 
 const getLocalTimestampInMilliSeconds = () => {
   return Date.now()
 }
 
-const fetchSystemTime = () => {
-  return metaphysics<SystemResults>({
-    query: `
-      {
+const fetchSystemTime = () =>
+  fetchQuery<TimeOffsetProviderQuery>(
+    defaultEnvironment,
+    graphql`
+      query TimeOffsetProviderQuery {
         system {
           time {
             unix
           }
         }
       }
     `,
-  })
-}
+    {},
+    { force: true }
+  )
 
 const getGravityTimestampInMilliSeconds = async () => {
   const startTime = getLocalTimestampInMilliSeconds()
-  const { data } = await fetchSystemTime()
+  const { system } = await fetchSystemTime()
 
   const possibleNetworkLatencyInMilliSeconds = (getLocalTimestampInMilliSeconds() - startTime) / 2
-  const serverTimestampInMilliSeconds = data.system.time.unix * 1e3 + possibleNetworkLatencyInMilliSeconds
+  const serverTimestampInMilliSeconds = system.time.unix * 1e3 + possibleNetworkLatencyInMilliSeconds
```

Comment:
cc @yuki24 @dleve123 Something to keep an eye on during next QA.

Reply:
Can you provide some more context here? Can you summarize the relevant diff, and provide perspective on any risk here?

Reply:
I changed this system time provider, used in bidding views, to use a different fetch mechanism. There should be _no_ functional differences, but still a good thing to go through the bidding views next QA (as you would normally do anyways).